PluginCore/ScintillaNet/Lexers.cs

```diff
@@ -80,9 +80,13 @@ public enum CPP
         TRIPLEVERBATIM = 21,
         HASHQUOTEDSTRING = 22,
         PREPROCESSORCOMMENT = 23,
-        WORD3 = 24,
-        WORD4 = 25,
-        WORD5 = 26,
+        PREPROCESSORCOMMENTDOC = 24,
+        USERLITERAL = 25,
+        TASKMARKER = 26,
+        ESCAPESEQUENCE = 27,
```

Comment:
Actually, it looks like the C++ escape sequences are not highlighted, it simply highlights `\` and the character after it.